data/assets/css/pages/_404.scss

```diff
@@ -0,0 +1,42 @@
+.error-container {
+  width: 100%;
+  text-align: center;
+  padding: 2rem 4rem;
+}
+
+.e404 {
+  font-weight: 900;
+  font-size: 13rem;
+}
+
+.complement {
+  font-size: 1.5rem;
+}
+
+@media screen and (max-width: 670px) {
```

Comment:
Thanks a lot for this pull request! But please sleep well :)

It is not big deal, but I have a change suggestion in this styles.
Current sass styles are "mobile first", and then there are two breakpoints for medium and large screens (you can see an example here https://github.com/VLCTechHub/VLCTechHub-site/blob/master/data/assets/css/pages/_event.scss#L32 or here https://github.com/VLCTechHub/VLCTechHub-site/blob/a6e70150326410852befa87199816ca6dc8a8e7a/data/assets/css/components/_event-summary.scss#L23) 

Could you change the styles so they use these general breakpoints?

(Just to let you know there are also some variables for common margin/padding values at https://github.com/VLCTechHub/VLCTechHub-site/blob/a6e70150326410852befa87199816ca6dc8a8e7a/data/assets/css/abstract/_variables.scss#L11, for 10px, 20px, ... in rem values in case you want to try them)